### ISSUE TITLE
Structure discourse

### ISSUE THREAD

**Issue by adamgayoso** [scverse/governance#25]

**Comment by Zethson** [comment]
> - Development
>    - Tag for each package

I get why you have this here, but was wondering whether we should try to keep development discussions inside Github issues?

It might be easier for everyone to have user and meta questions/topics in the discourse and development issues in Github issues? Else, we might have development discussions happening in two places and it might get hard to track it?

We could then also more easily point people to the appropriate place to post their bug or help request.

**Comment by adamgayoso** [comment]
> I get why you have this here, but was wondering whether we should try to keep development discussions inside Github issues?

Yes, can probably remove this card. I was thinking like making a new model development with scvi-tools, which can have it's own subcategory in the help card.

**Comment by ivirshup** [comment]
I'm not sure I understand the purpose of the assays topic.

Are you trying to separate questions into "basic help" and "higher level questions"?

Would an "Ideas" topic make more sense? It seems like higher level questions wouldn't be limited just to assays. I would also suspect there would be overlap between "assays" and "help".

**Comment by ivirshup** [comment]
I think the distinction makes sense.

I think the name "Assay" may end up drawing usage questions for particular assays which may fit better in "Help". I think "Help" may also attract some higher level questions.

What do you think about naming this something different? Like "Ideas", "Discussion", "Questions"?

**Comment by adamgayoso** [comment]
"Ideas" seems to broad to me because like they are questions not ideas. We could call it "general"?

Everything is generally ready except domain change. I made every core person who signed up a level 4 user (basically same as moderator, but we can only have 5 moderators). @ivirshup can we sync up to change the domain name?

https://discourse.scvi-tools.org/

**Comment by ivirshup** [comment]
I think "general" is good (but broader than "ideas", no?)
